### PR TITLE
Beta docs restructure ctl plane services

### PR DESCRIPTION
**Pull request by klgill**
Part 1 of restructuring the "adopting services" procedures.

Writers: Please use the peer review checklist: https://docs.google.com/document/d/1hxKJdhGNqSOTg6WrbYjJaQIWDu7x5cl2QRYJ7G0HFt8/edit#heading=h.o8ezt27iw7kl

### CODE REVIEW

**Review by fao89 — APPROVED** [review]

**Review by jistr — APPROVED** [review]

File: docs_user/modules/proc_deploying-backend-services.adoc
Comment (rheslop) [inline docs_user/modules/proc_deploying-backend-services.adoc:1]: Needs ID

File: docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc
Comment (rheslop) [inline docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc:1]: Needs ID

File: docs_user/modules/proc_adopting-the-image-service.adoc [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
-Adopting Glance means that an existing `OpenStackControlPlane` CR, where Glance
+Adopting {image_service_first_ref} means that an existing `OpenStackControlPlane` custom resource (CR), where {image_service}
 is supposed to be disabled, should be patched to start the service with the
 configuration parameters provided by the source environment.
 
 When the procedure is over, the expectation is to see the `GlanceAPI` service
-up and running: the `Keystone endpoints` should be updated and the same backend
-of the source Cloud will be available. If the conditions above are met, the
-adoption is considered concluded.
+up and running: the {identity_service} endpoints should be updated and the same backend of the source Cloud will be available. If the conditions above are met, the adoption is considered concluded.
```
Comment (rheslop) [inline docs_user/modules/proc_adopting-the-image-service.adoc:13]: up and running: the {identity_service} endpoints are updated and the same backend of the source Cloud is available. If the conditions above are met, the adoption is considered concluded.

Comment (rheslop) [reply docs_user/modules/proc_adopting-the-image-service.adoc:13]: Changed to present tense

File: docs_user/modules/proc_adopting-the-image-service.adoc [hunk 2]
```diff
@@ -1,40 +1,36 @@
...
+//This link goes to a 404. Do we need this text downstream? 
 As already done for https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/keystone_adoption.md[Keystone], the Glance Adoption follows the same pattern.
 
-=== Using Swift backend
+.Using {object_storage} backend
 
-When Glance is deployed with Swift as a backend in the source environment based
-on TripleO, the podified `glanceAPI` instance should be deployed with the following
+When {image_service} is deployed with {object_storage_first_ref} as a backend in the source environment based
+on TripleO, the control plane `glanceAPI` instance should be deployed with the following
```
Comment (rheslop) [inline docs_user/modules/proc_adopting-the-image-service.adoc:33]: Replace 'should be' with 'is'

File: docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 = Creating a Ceph NFS cluster
 
-If you use the Ceph via NFS backend with {rhos_component_storage_file_first_ref}, prior to adoption, you must create a new clustered NFS service on the Ceph cluster. This service will replace the standalone, pacemaker-controlled `ceph-nfs` service that was used on {osp_prev_long} {osp_prev_ver}.
+If you use the Ceph via NFS backend with {rhos_component_storage_file_first_ref}, prior to adoption, you must create a new clustered NFS service on the Ceph cluster. This service will replace the standalone, pacemaker-controlled `ceph-nfs` service that was used on {rhos_prev_long} {rhos_prev_ver}.
```
Comment (rheslop) [inline docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc:5]: I don't see the `rhos_component_storage_file_first_ref ` attribute in the current 18.0 branch

File: docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc [hunk 4]
```diff
@@ -6,7 +6,7 @@
 = Migrating databases to MariaDB instances
 
 This document describes how to move the databases from the original
-{osp_prev_long} ({OpenStackShort}) deployment to the MariaDB instances in the {OpenShift} cluster.
+{rhos_prev_long} ({OpenStackShort}) deployment to the MariaDB instances in the {OpenShift} cluster.
```
Comment (rheslop) [inline docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc:9]: Is OpenStackShort attribute in another file other than rhoso_attributes?

File: docs_user/modules/openstack-pull_openstack_configuration.adoc
[the remark this answers is not in the document]
Comment (klgill) [reply docs_user/modules/openstack-pull_openstack_configuration.adoc:1]: This file is renamed in the following PR that is not merged yet: https://github.com/openstack-k8s-operators/data-plane-adoption/pull/402/files

File: docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc (hunk 4 above)
Comment (klgill) [reply docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc:9]: Yes, it's in the adoption-attributes file.

File: docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc (hunk 3 above)
Comment (klgill) [reply docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc:5]: It's not there yet. I need to copy the attributes I used upstream to the downstream adoption-attributes.adoc.

**Pull request merged**